test/blackbox-tests/test-cases/cxx-flags.t/run.t

```diff
@@ -47,9 +47,9 @@ With use_standard_c_and_cxx_flags = true
   > EOF
 
 > Check that compiler detection is done
-  $ dune build .dune/ccomp
+  $ dune build .dune/ccomp/ccomp
 
-  $ cat _build/default/.dune/ccomp |
+  $ cat _build/default/.dune/ccomp/ccomp |
```

Comment:
Is this change intentional?

Reply:
Yeah, I changed the code producing the configurator files to use normal build rules. To force them to be generated eagerly, `gen_rules` requests them in every directory. That created a cycle for these rules, so I moved them in a sub-directory.

Reply:
BTW, we can now use `Action_builder.action_stdout` for such cases